react/week4/src/GithubList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+
+function fetchGithub(query) {
+    const URL = `https://api.github.com/search/users?q=${query}`
+    return fetch(URL).then((response) => response.json())
+}
+
+export default class GithubList extends React.Component {
+    state = {
+        users: [], 
+        isLoading: true, 
+        searchUser: '',
+        errorMessage: null
+    }
+
+
+handleSubmit = (e) => {
```

Comment:
I think that the indentation may be incorrect here. Shouldn't this be one level deeper (i.e. 4 spaces)?